src.gen/@amzn/amazon-q-developer-streaming-client/package.json

```diff
@@ -0,0 +1,91 @@
+{
+    "name": "@amzn/amazon-q-developer-streaming-client",
```

Comment:
What is the purpose of this package? Why can we not use `src.gen/@amzn/codewhisperer-streaming`?

Reply:
To answer this, please update https://github.com/aws/aws-toolkit-vscode/blob/master/docs/build.md

Reply:
As discussed offline, this client supports Sigv4 credentials instead of bearer token. this is mandatory to support both IAM and SSO users.

The instructions to generate these clients will be updated in the same quip doc referenced in this .md file
I updated the wording to reflect the new changes

Reply:
The quip wasn't updated. I added a "qdevstreaming" section.